addOns/postman/src/main/java/org/zaproxy/addon/postman/models/Body.java

```diff
@@ -154,39 +154,6 @@ public void setSrc(String src) {
         }
     }
 
-    @JsonIgnoreProperties(ignoreUnknown = true)
-    public static class FormData extends KeyValueData {
```

Comment:
Had to remove this, jackson was having problems in deciding between `KeyValueData` and `FormData`. Seems like type deduction doesn't work with inheritance when base class is not abstract.

Reply:
Not sure if related but `ObjectDeserializer` should also use the original configuration/`ObjectMapper` rather than create a new one (as it will be hitting the class loading issue as well).

Reply:
Okay, will raise another PR for this. For this one I tried with adding explicit dependency and still had the problem which I stated.